imod/tests/fixtures/flow_transport_simulation_fixture.py

```diff
@@ -157,7 +154,7 @@ def flow_transport_simulation():
         concentration_boundary_type="Aux",
         screen_top=[0.0, 0.0],
         screen_bottom=[-1.0, -1.0],
-        rate=[1.0, -2.0],
+        rate=[0.1, -0.2],
```

Comment:
Please add a comment how and why we departed from the original mf6 example

Reply:
done. note that the fixture did not mention the original mf6 example, so nothing was promised wrt that. 
I added a paragraph to explain where the model in the fixture comes from and why it was changed.